lessons/Early-Math/equivalence/langs/en-us/index.adoc

```diff
@@ -0,0 +1,166 @@
+= Equivalence and Variables
+
+@description{Students use properties to analyze when equations both with and without variables are true.}
+
+@lesson-prereqs{associative-property}
+
+@keywords{variable}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Determine if two expressions are equivalent.
+* Analyze Circles to determine if equivalency exists on account of the Associative Property, the Commutative Property, or Computation.
+
+
+| Student-facing Lesson Goals
+|
+
+* Let's use Circles of Evaluation to decide if expressions with vairables are equivalent.
+
+
+| Materials
+|[.materials-links]
+@material-links
+
+
+| Key Points For The Facilitator
+|
+* This lesson introduces variables. The cognitive leap required when moving from operations with numbers to operations with variables is a big one! In this lesson, Circles of Evaluation can help ease the transition.
+|===
+
+== Seeing Equivalence @duration{25 minutes}
+
+=== Overview
+Students analyze Circles to determine if two expressions are equivalent or not, applying the Commutative Property, the Associative Property, and computation.
+
+=== Launch
+As we've observed, Circles and laws of arithmetic can help us understand whether an equation is true or false.
+
+Invite students to analyze Equation 1, represented by these two Circles:
+
+[.embedded, cols="^.^3,^.^1,^.^3", grid="none", stripes="none" frame="none"]
+|===
+|@show{(coe '(- (* 22 3) 4))}	| @math{=} | @show{(coe '(- (* 3 22) 4))}
+|===
+
+[.lesson-instruction]
+- What do you Notice? What do you Wonder?
+** _Possible responses: The outer circle subtracts @math{4}, the inner circle multiplies. @math{22} and @math{3} swap positions._
+
+Next, have students look at Equation 2, represented by these two Circles.
+
+[.embedded, cols="^.^3,^.^1,^.^3", grid="none", stripes="none" frame="none"]
+|===
+|@show{(coe '(- (* 22 3) 4))}	| @math{=} | @show{(coe '(- 4 (* 22 3)))}
+|===
+
+[.lesson-instruction]
+- Here, what do you Notice? What do you Wonder?
+** _Possible responses: The same numbers and operations are included. This time, the product (or the inner circle) and @math{4} swap positions._
+- Which is true: Equation 1, Equation 2, or both? How do you know?
+** _The first equation is true. We see the Commutative Property of multiplication being applied when @math{22 \times 3} is transformed to @math{3 \times 22}._
+** _In the second equation, it looks as though there has been a misguided attempted to apply the Commutative Property; this does not work because the Commutative Property does not hold for subtraction._
+** _We can also use Computation to verify that both sides of the first equation equal @math{62}. The second equation's left side is @math{62}, but its right side is @math{-62}._
+
+As students observe and share out, urge them to justify their ideas about equivalence.
+
+=== Investigate
+
```

Comment:
Suggest overhauling these instructions so that they are something more in the direction of:

We have learned that computation is an effective strategy for determining whether or not two expressions are equivalent. Pretty soon we'll be looking at expressions with variables, where our computation strategy won't work as well. Let's see if we can figure out how to determine whether or not two expressions are equivalent without computation! For now, while we're still looking at expressions with numbers, we can doublecheck our thinking with computation.